src/api/spec/support/vcr.rb

```diff
@@ -7,7 +7,7 @@
   config.default_cassette_options = { record: :once }
   config.allow_http_connections_when_no_cassette = true
   config.configure_rspec_metadata!
-  # config.debug_logger = File.open(Rails.root.join('log', 'vcr.log'), 'w')
+  config.debug_logger = File.open(Rails.root.join('log', 'vcr.log'), 'w')
```

Comment:
Left from debugging?

Reply:
github isn't able to identify the changes and collapse the outdated comments anymore?